rust/Earthfile

```diff
@@ -1,24 +1,45 @@
 VERSION --global-cache 0.7
 
+# INIT stores the configuration required for the other UDCs in the filesystem, and installs required dependencies.
+#   - cache_id: Overrides default ID of the global $CARGO_HOME cache. Its value is exported to the build environment under the entry: $CARGO_HOME_CACHE_ID
+#   - keep_fingerprints (false): Instructs the following +CARGO calls to don't remove the Cargo fingerprints of the source packages. Use only when source packages have been COPYed with --keep-ts option.
+#   - sweep_days (4): +CARGO calls use cargo-sweep to clean build artifacts that haven't been accessed for this number of days.
+INIT:
+    COMMAND
+    DO +INSTALL_CARGO_SWEEP
+    RUN mkdir -p /earthly/cfg
+
+    # cache_id
+    ARG EARTHLY_TARGET_PROJECT_NO_TAG
+    ARG OS_RELEASE=$(md5sum /etc/os-release | cut -d ' ' -f 1)
+    ARG cache_id="${EARTHLY_TARGET_PROJECT_NO_TAG}#${OS_RELEASE}#earthly-cargo-cache"
```

Comment:
You might want to use `SET` here instead of `ARG`, which will indicate this is not something that can/should be overridden (unless I'm misunderstanding it).

Reply:
Good point, I tried. But found that `--arg-scope-and-set` needs to be set in the calling Earthfile (that seems like a bug), and also builtin args cannot be set via `LET` (that looks more appropriate), so decide to leave it for now

Reply:
@vladaionescu any thoughts on this?

Reply:
https://github.com/earthly/earthly/issues/3490

Reply:
Leaving it SGTM